pkg/logger/logger.go

```diff
@@ -283,6 +283,21 @@ func (l *Logger) Sync() error {
 	return l.l.Sync()
 }
 
+// Setters
+// TODO: add SetWriter and Setlogger to the struct level?
```

Comment:
logger offers two ways of logging... by the package level and by receiver level. In the tracee we are only using the pkg level. For the sake of compliance I would add the option for receivers. If in the future we are not going to use different loggers (instantiations), we can keep the pkg level by just removing the rest.

Reply:
My initial solution was first implementing the receiver then using it globally, however it didn't work as expected. I don't think we actually use the receiver level at all right? Maybe we should consider removing it entirely.
(note: trivy also uses a similar strategy with only a global logger they initialize once with an InitGlobal like method. It would simplify our logger a bit to be similar to that).

Reply:
> I don't think we actually use the receiver level at all right? Maybe we should consider removing it entirely.

I agree. I just implemented it as a option since we didn't have a specific goal that time. So let's remove the receiver option.

> (note: trivy also uses a similar strategy with only a global logger they initialize once with an InitGlobal like method. It would simplify our logger a bit to be similar to that).

Alright.